tests/test_routes.py

```diff
@@ -25,7 +25,7 @@
 from wsgi import app
 from service.common import status
 from service.models import db, Order
-from tests.factories import OrderFactory, ItemFactory
```

Comment:
Why remove ItemFactory?

Reply:
This was done because we removed the test case from this file, and so ItemFactory was not really used elsewhere